assessment.rb

```diff
@@ -2,11 +2,31 @@
 # Assessment answers to:
 
 # question 2:
+licensee = Licensee.new surname: 'Bishop', given_name: 'Caren', gender: 'f', height: 63, weight: 132, dob: '1943-09-26' , eye_color: 'Brown', hair_color: 'Black'
+p licensee
 
 # question 3:
+Licensee.transaction do
+  CSV.foreach(Rails.root + 'data/licensees.csv') do |row|
+    licensee = row.to_hash
```

Comment:
You've taken the data from the row, but you have't created any new Licensee instances yet!